### PR TITLE
Added How To page for Provision an Ubuntu virtual machine running SQL Server in Azure and  AWS Pro activation 

### PR DESCRIPTION
**Pull request by billingsmoore**
Addresses "Public Cloud: Write a how-to guide for Azure based on an existing tutorial #48" from Open Documentation Academy.

Added How To page for Provision an Ubuntu virtual machine running SQL Server in Azure based on this tutorial:
https://ubuntu.com/tutorials/provision-an-ubuntu-virtual-machine-running-sql-server-in-azure#2-launch-an-ubuntu-vm-with-sql-server-installed

Added new page to index

### CODE REVIEW

File: aws/aws-how-to/instances/upgrade-to-ubuntu-pro-at-scale-using-tokens-with-ssm.rst
Comment (codyshepherd) [inline aws/aws-how-to/instances/upgrade-to-ubuntu-pro-at-scale-using-tokens-with-ssm.rst:77]: This sentence needs re-wording. "The token will be requested to the user" is unclear. Also unclear is what is meant by "it" in "then it will execute the rest of the code."

File: aws/aws-how-to/instances/upgrade-to-ubuntu-pro-at-scale-using-tokens-with-ssm.rst
Comment (codyshepherd) [inline aws/aws-how-to/instances/upgrade-to-ubuntu-pro-at-scale-using-tokens-with-ssm.rst:79]: What kind of update? APT update? Can you add an example command here?

File: aws/aws-how-to/instances/upgrade-to-ubuntu-pro-at-scale-using-tokens-with-ssm.rst
Comment (codyshepherd) [inline aws/aws-how-to/instances/upgrade-to-ubuntu-pro-at-scale-using-tokens-with-ssm.rst:80]: How does one attach the token?

File: aws/aws-how-to/instances/upgrade-to-ubuntu-pro-at-scale-using-tokens-with-ssm.rst
Comment (codyshepherd) [inline aws/aws-how-to/instances/upgrade-to-ubuntu-pro-at-scale-using-tokens-with-ssm.rst:89]: This might be a good place to describe what "the SSM script" is

File: aws/aws-how-to/instances/upgrade-to-ubuntu-pro-at-scale-using-tokens-with-ssm.rst
Comment (codyshepherd) [inline aws/aws-how-to/instances/upgrade-to-ubuntu-pro-at-scale-using-tokens-with-ssm.rst:91]:
```suggestion
You don’t need to restart the machines after applying the token. If the SSM script fails, check the SSM format of your document, see if there are any typos and try again. If this does not work check the output generated by SSM and look for any configuration issue (do your machines have access to update and upgrade packages? Pro repositories are available on different endpoints [[than what?]]. Check the references at the end of the tutorial).
```

What endpoints are Pro repositories available at? Are these static, and if so, can they be listed here?

File: aws/aws-how-to/instances/upgrade-to-ubuntu-pro-at-scale-using-tokens-with-ssm.rst
Comment (codyshepherd) [inline aws/aws-how-to/instances/upgrade-to-ubuntu-pro-at-scale-using-tokens-with-ssm.rst:98]:
```suggestion
Check the Pro status of your machine
```

File: aws/aws-how-to/instances/upgrade-to-ubuntu-pro-at-scale-using-tokens-with-ssm.rst
Comment (codyshepherd) [inline aws/aws-how-to/instances/upgrade-to-ubuntu-pro-at-scale-using-tokens-with-ssm.rst:107]:
```suggestion
This will display your entitlements and enabled services.
```

File: azure/azure-how-to/instances/provision-an-ubuntu-virtual-machine-running-sql-server-in-azure.rst
Comment (codyshepherd) [inline azure/azure-how-to/instances/provision-an-ubuntu-virtual-machine-running-sql-server-in-azure.rst:27]: Out of curiosity, why is Focal being used in this example?

Comment (codyshepherd) [inline azure/azure-how-to/instances/provision-an-ubuntu-virtual-machine-running-sql-server-in-azure.rst:27]:
```suggestion
- Select the image by the name “SQL 2019 on Ubuntu Pro 20.04 LTS with 24x7 Support”.
```

File: azure/azure-how-to/instances/provision-an-ubuntu-virtual-machine-running-sql-server-in-azure.rst
Comment (codyshepherd) [inline azure/azure-how-to/instances/provision-an-ubuntu-virtual-machine-running-sql-server-in-azure.rst:35]: Is this a menu title or does the user have to know their Subscription and Resource Group ID?

File: azure/azure-how-to/instances/provision-an-ubuntu-virtual-machine-running-sql-server-in-azure.rst
Comment (codyshepherd) [inline azure/azure-how-to/instances/provision-an-ubuntu-virtual-machine-running-sql-server-in-azure.rst:71]: Isn't the default username for ubuntu cloud images `ubuntu`?

File: azure/azure-how-to/instances/provision-an-ubuntu-virtual-machine-running-sql-server-in-azure.rst
Comment (codyshepherd) [inline azure/azure-how-to/instances/provision-an-ubuntu-virtual-machine-running-sql-server-in-azure.rst:76]: What is `SA`?

File: azure/azure-how-to/instances/provision-an-ubuntu-virtual-machine-running-sql-server-in-azure.rst
Comment (codyshepherd) [inline azure/azure-how-to/instances/provision-an-ubuntu-virtual-machine-running-sql-server-in-azure.rst:98]:
```suggestion
Several SQL Server packages are installed by default, including the SQL Server command-line tools package. The tools package contains the `sqlcmd` and `bcp` tools. For convenience, you can add the tools path, `/opt/mssql-tools/bin/`, to your PATH environment variable.
```

**Review by codyshepherd — CHANGES_REQUESTED** [review]
The title of this PR should be changed to reflect that steps for AWS Pro activation are included as well

File: aws/aws-how-to/instances/upgrade-to-ubuntu-pro-at-scale-using-tokens-with-ssm.rst
Comment (billingsmoore) [reply aws/aws-how-to/instances/upgrade-to-ubuntu-pro-at-scale-using-tokens-with-ssm.rst:79]: This is handled by the SSM document itself. I've rephrased to clarify that.

File: aws/aws-how-to/instances/upgrade-to-ubuntu-pro-at-scale-using-tokens-with-ssm.rst
Comment (billingsmoore) [reply aws/aws-how-to/instances/upgrade-to-ubuntu-pro-at-scale-using-tokens-with-ssm.rst:89]: The SSM script is described in the previous step.

File: azure/azure-how-to/instances/provision-an-ubuntu-virtual-machine-running-sql-server-in-azure.rst
Comment (billingsmoore) [reply azure/azure-how-to/instances/provision-an-ubuntu-virtual-machine-running-sql-server-in-azure.rst:71]: Do you think the example ssh command should use 'ubuntu@20.55.55.555 instead?

File: aws/aws-how-to/instances/upgrade-to-ubuntu-pro-at-scale-using-tokens-with-ssm.rst
Comment (billingsmoore) [reply aws/aws-how-to/instances/upgrade-to-ubuntu-pro-at-scale-using-tokens-with-ssm.rst:24]: I don't know the answer to this. I am not the original author of the tutorial, I've just adapted it for style as requested in Issue #48 "Public Cloud: Write a how-to guide for Azure based on an existing tutorial https://github.com/canonical/ubuntu-cloud-docs/pull/48" from Open Documentation Academy.

I'm happy to make adjustments here if you can direct me to where I could find the answer to this sort of question.

File: azure/azure-how-to/instances/provision-an-ubuntu-virtual-machine-running-sql-server-in-azure.rst
Comment (billingsmoore) [reply azure/azure-how-to/instances/provision-an-ubuntu-virtual-machine-running-sql-server-in-azure.rst]: This is the example from the original tutorial. I've left examples exactly as they appeared there.

File: aws/aws-how-to/instances/upgrade-to-ubuntu-pro-at-scale-using-tokens-with-ssm.rst
Comment (billingsmoore) [reply aws/aws-how-to/instances/upgrade-to-ubuntu-pro-at-scale-using-tokens-with-ssm.rst]: I don't know the exact definition for machines here. This phrasing is from the original tutorial and is also the phrasing used in announcements such as this: https://ubuntu.com/blog/ubuntu-pro-beta-release

File: azure/azure-how-to/instances/provision-an-ubuntu-virtual-machine-running-sql-server-in-azure.rst
Comment (danpdraper) [inline azure/azure-how-to/instances/provision-an-ubuntu-virtual-machine-running-sql-server-in-azure.rst:92]: This should be a code block.

File: azure/azure-how-to/instances/provision-an-ubuntu-virtual-machine-running-sql-server-in-azure.rst
Comment (danpdraper) [inline azure/azure-how-to/instances/provision-an-ubuntu-virtual-machine-running-sql-server-in-azure.rst:98]: `PATH` (code)

File: azure/azure-how-to/instances/provision-an-ubuntu-virtual-machine-running-sql-server-in-azure.rst
Comment (danpdraper) [inline azure/azure-how-to/instances/provision-an-ubuntu-virtual-machine-running-sql-server-in-azure.rst:100]: `PATH` (code)

File: azure/azure-how-to/instances/provision-an-ubuntu-virtual-machine-running-sql-server-in-azure.rst
Comment (danpdraper) [inline azure/azure-how-to/instances/provision-an-ubuntu-virtual-machine-running-sql-server-in-azure.rst:104]: This should be a code block.

File: azure/azure-how-to/instances/provision-an-ubuntu-virtual-machine-running-sql-server-in-azure.rst
Comment (danpdraper) [inline azure/azure-how-to/instances/provision-an-ubuntu-virtual-machine-running-sql-server-in-azure.rst:110]: You're missing a period after the second sentence.

File: azure/azure-how-to/instances/provision-an-ubuntu-virtual-machine-running-sql-server-in-azure.rst
Comment (danpdraper) [inline azure/azure-how-to/instances/provision-an-ubuntu-virtual-machine-running-sql-server-in-azure.rst:116]: `"Add inbound port"`

`"Inbound Port Rules"`

File: azure/azure-how-to/instances/provision-an-ubuntu-virtual-machine-running-sql-server-in-azure.rst
Comment (danpdraper) [inline azure/azure-how-to/instances/provision-an-ubuntu-virtual-machine-running-sql-server-in-azure.rst:118]: `"MS SQL"`

File: azure/azure-how-to/instances/provision-an-ubuntu-virtual-machine-running-sql-server-in-azure.rst
Comment (danpdraper) [inline azure/azure-how-to/instances/provision-an-ubuntu-virtual-machine-running-sql-server-in-azure.rst:129]: `In a terminal` would be more appropriate than `In the BASH shell`.

File: azure/azure-how-to/instances/provision-an-ubuntu-virtual-machine-running-sql-server-in-azure.rst
Comment (danpdraper) [inline azure/azure-how-to/instances/provision-an-ubuntu-virtual-machine-running-sql-server-in-azure.rst:131]: This should be a code block.

File: azure/azure-how-to/instances/provision-an-ubuntu-virtual-machine-running-sql-server-in-azure.rst
Comment (danpdraper) [inline azure/azure-how-to/instances/provision-an-ubuntu-virtual-machine-running-sql-server-in-azure.rst:137]: You're missing a period.

**Review by danpdraper — CHANGES_REQUESTED** [review]

File: aws/aws-how-to/instances/upgrade-to-ubuntu-pro-at-scale-using-tokens-with-ssm.rst
Comment (codyshepherd) [reply aws/aws-how-to/instances/upgrade-to-ubuntu-pro-at-scale-using-tokens-with-ssm.rst:24]: I would at least add something like "an AWS account with write access to SSM and the ability to connect to VMs [via Fleet Manager]". It seems like those are the two main pieces of functionality required at the AWS level.

File: azure/azure-how-to/instances/provision-an-ubuntu-virtual-machine-running-sql-server-in-azure.rst
Comment (billingsmoore) [reply azure/azure-how-to/instances/provision-an-ubuntu-virtual-machine-running-sql-server-in-azure.rst:102]: I'm sorry, I don't understand what this means.

File: azure/azure-how-to/instances/provision-an-ubuntu-virtual-machine-running-sql-server-in-azure.rst
Comment (danpdraper) [reply azure/azure-how-to/instances/provision-an-ubuntu-virtual-machine-running-sql-server-in-azure.rst:104]: See above.

**Review by danpdraper — APPROVED** [review]

File: aws/aws-how-to/instances/upgrade-to-ubuntu-pro-at-scale-using-tokens-with-ssm.rst
Comment (codyshepherd) [inline aws/aws-how-to/instances/upgrade-to-ubuntu-pro-at-scale-using-tokens-with-ssm.rst:6]: RST is very picky about the length of these underlines matching the length of the word they underline. Please fix these so the CI jobs can pass.

File: aws/aws-how-to/instances/upgrade-to-ubuntu-pro-at-scale-using-tokens-with-ssm.rst
Comment (codyshepherd) [inline aws/aws-how-to/instances/upgrade-to-ubuntu-pro-at-scale-using-tokens-with-ssm.rst:31]:
```suggestion
------------
```
```suggestion
--------------
```

File: aws/aws-how-to/instances/upgrade-to-ubuntu-pro-at-scale-using-tokens-with-ssm.rst
Comment (codyshepherd) [inline aws/aws-how-to/instances/upgrade-to-ubuntu-pro-at-scale-using-tokens-with-ssm.rst:6]:
```suggestion
---------
```
```suggestion
--------
```

File: aws/aws-how-to/instances/upgrade-to-ubuntu-pro-at-scale-using-tokens-with-ssm.rst
Comment (codyshepherd) [inline aws/aws-how-to/instances/upgrade-to-ubuntu-pro-at-scale-using-tokens-with-ssm.rst:42]:
```suggestion
------------------------------------------
```
```suggestion
--------------------------------------------
```

File: aws/aws-how-to/instances/upgrade-to-ubuntu-pro-at-scale-using-tokens-with-ssm.rst
Comment (codyshepherd) [inline aws/aws-how-to/instances/upgrade-to-ubuntu-pro-at-scale-using-tokens-with-ssm.rst:87]:
```suggestion
------------------
```
```suggestion
---------------------
```

File: aws/aws-how-to/instances/upgrade-to-ubuntu-pro-at-scale-using-tokens-with-ssm.rst
Comment (codyshepherd) [inline aws/aws-how-to/instances/upgrade-to-ubuntu-pro-at-scale-using-tokens-with-ssm.rst:99]:
```suggestion
----------------
```
```suggestion
---------------------------------------
```

File: azure/azure-how-to/instances/provision-an-ubuntu-virtual-machine-running-sql-server-in-azure.rst
Comment (codyshepherd) [inline azure/azure-how-to/instances/provision-an-ubuntu-virtual-machine-running-sql-server-in-azure.rst:34]:
```suggestion
~~~~~~~~~~~~~~
```
```suggestion
~~~~~~~~~~~~~
```

File: azure/azure-how-to/instances/provision-an-ubuntu-virtual-machine-running-sql-server-in-azure.rst
Comment (codyshepherd) [inline azure/azure-how-to/instances/provision-an-ubuntu-virtual-machine-running-sql-server-in-azure.rst:68]:
```suggestion
~~~~~~~~~~~~~~~~~~
```
```suggestion
~~~~~~~~~~~~~~~~~
```

File: azure/azure-how-to/instances/provision-an-ubuntu-virtual-machine-running-sql-server-in-azure.rst
Comment (codyshepherd) [inline azure/azure-how-to/instances/provision-an-ubuntu-virtual-machine-running-sql-server-in-azure.rst:79]:
```suggestion
~~~~~~~~~~~~~~~~~~~~~~~
```
```suggestion
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

**Review by codyshepherd — CHANGES_REQUESTED** [review]
Please correct the heading formatting

File: aws/aws-how-to/instances/index.rst
Comment (codyshepherd) [inline aws/aws-how-to/instances/index.rst:49]:
```suggestion
   Upgrade to Ubuntu Pro at scale using tokens with SSM <upgrade-to-ubuntu-pro-at-scale-using-tokens-with-ssm>
   Configure automated updates <automatically-update-ubuntu-instances>
```

**Review by codyshepherd — CHANGES_REQUESTED** [review]
Looks like there's one more thing: the article needs added to the [toctree](https://documatt.com/restructuredtext-reference/element/toctree.html)

**Comment by codyshepherd** [comment]
CI is now throwing a spelling error because of the word `ssm`. That string can be added to `.custom_wordlist.txt`  to resolve the error.

**Comment by codyshepherd** [comment]
There are additional formatting errors, including words not in the custom wordlist and incorrect usage of `.. code-block::` blocks, which need a blank line afterward. You can run tests locally via make commands, such as `make install` followed by `PROJECT=azure make spelling`. Once the CI passes I'll be happy to approve.

**Review by codyshepherd — APPROVED** [review]
Thanks for pushing through all the finnicky edits.

**Comment by billingsmoore** [comment]
> Thanks for pushing through all the finnicky edits.

Thanks for the patience on your end!

**Pull request merged**
